skll/learner/utils.py

```diff
@@ -282,10 +285,11 @@ def compute_evaluation_metrics(metrics,
     # initialize a dictionary that will hold all of the metric scores
     metric_scores = {metric: None for metric in metrics}
 
-    # if we are a classifier and in probability mode, then
-    # `yhat` are probabilities so we need to compute the
-    # class indices separately and save them too
-    if probability and model_type == 'classifier':
+    # if we are doing classification and are a probablistic
+    # learner or a soft-voting meta learner, then `yhat` are
+    # probabilities so we need to compute the  class indices
```

Comment:
If you are making other changes to this or other files:

```suggestion
    # probabilities so we need to compute the class indices
```